requirements27.txt

```diff
@@ -0,0 +1,8 @@
+requests
```

Comment:
because the uvicorn dependency is not python 2 compatible; to avoid throwing exceptions or errors, we needed to create a python 2 specific version of our requirements.txt file